structure_tests/drivers/docker_driver.go

```diff
@@ -311,3 +305,32 @@ func (d *DockerDriver) exec(t *testing.T, env []string, command []string) (strin
 
 	return stdout.String(), stderr.String(), exitCode
 }
+
+func (d *DockerDriver) GetConfig(t *testing.T) (unversioned.Config, error) {
+	img, err := d.cli.InspectImage(d.currentImage)
+	if err != nil {
+		t.Errorf("Error when inspecting image: %s", err.Error())
+		return unversioned.Config{}, err
+	}
+
+	// docker provides these as maps (since they can be mapped in docker run commands)
```

Comment:
Just making sure I understand, this is a map of what to what? volume name to mountpoint?

Reply:
Could you link to a struct that shows what type Volumes is? I'm having trouble finding it since we unmarshall it from json.

Reply:
I'm actually not sure if docker has any real documentation on this, which is annoying. I did some trial and error here to figure it out: it's a map of the exposed volume directory in the image to the mountpoints on the host. Since users can only specify host mountpoints through docker run commands (doesn't make sense through a dockerfile), we don't actually care about those here though.

https://godoc.org/github.com/fsouza/go-dockerclient#Config shows the `Volumes` being returned in a `map[string]struct{}`, which comes directly from the docker API.